### PR TITLE
minor additions/edits

### PR DESCRIPTION
**Pull request by malloc3**
Not sure we want to merge this.  Only changes the Debug library and I realize now that you may have intentionally include the Sample Id's for some reason.

### CODE REVIEW

File: diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb [hunk 1]
```diff
@@ -20,13 +20,19 @@ def build_stripwell_primer_probe_compositions(operations:)
...
-      stripwells = find_stripwells(sample_names: options[:sample_names],
-                                   stripwell_ot: options[:object_type],
-                                   rows: op.temporary[:microtiter_plate]
-                                                 .collection.dimensions[0])
-      if stripwells.any?(&:empty?)
-        op.error(ProtocolError, 'No enough parts in inventory')
+      given_stripwells = options[:stripwell_id]
+      stripwells = []
+      if given_stripwells
+        stripwells = given_stripwells.map { |id| Collection.find(id) }
```
Comment (dvnstrcklnd) [inline diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb]: Should be able to do `Collection.find(given_stripwells)`

Comment (malloc3) [reply diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb]: Done

**Comment by dvnstrcklnd** [comment]
Oh I just realized it says you're not sure if you want to merge. Should we revert?

**Comment by malloc3** [comment]
We should discuss.   I changed some of the debugging parts from listing Item ID's.    And then I realized that you had added the ID's and didn't want to undo that if you had a good reason.

**Pull request merged**